styles/common.css

```
@@ -0,0 +1,125 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap');
+
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:opsz,wght@9..40,300;9..40,400;9..40,500;9..40,700&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    justify-content: center;
+}
+
+.container {
+    width: 1000vh;
+    max-width: 1204px;
+}
+
+.navigation {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 230px;
+    font-family: DM Sans;
+    font-size: 20px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 26px;
+}
+
+.logo-wrap img {
+    width: 97px;
+    height: 59px;
+    margin-top: 41px;
+}
+
+.navigation-header {
+    display: flex;
+    align-items: center;
+    margin-top: 41px;
+}
+
+.navigation-header a {
+    margin-right: 24px;
+    text-decoration: none;
+    color: #666;
+    text-align: center;
+    font-family: DM Sans;
```

Outra boa prática é colocar fonts alternativas caso a font principal não carrege por algum motivo.

Por exemplo, nesse caso se a font **DM Sans** não carregar ele vai tentar carregar a **Arial**, se a **Arial** não carregar e tenta a **Helvetica** e assim por diante.
```suggestion
    font-family: DM Sans, Arial, Helvetica, sans-serif;
```

---

A ideia é colocar fonte de backup em todos os css's né isso? Só pra alivio de conciencia...

---

Isso, em todos os lugares que vc tá usando `font-family`

---

Feito!